R/likelihood_data.R

```diff
@@ -154,6 +154,9 @@ g3l_likelihood_data <- function (nll_name, data, missing_val = 0, area_group = N
     if (!('year' %in% names(data))) {
         stop("Data must contain a year column")
     }
+    if ('step' %in% names(data) & any(data$step > 99)) {
+        stop("The number of steps per year cannot exceed 99")
+    }
```

Comment:
This might make more sense at the top of ``g3s_time_convert``(), since then it's right next to the code that needs this. But I'm just making comments to avoid work now :)